pynamodb/attributes.py

```diff
@@ -77,8 +77,18 @@ def _initialize_attributes(cls):
         """
         cls._attributes = {}
         cls._dynamo_to_python_attrs = {}
-        for item_name, instance in six.iteritems(cls.__dict__):
-            if issubclass(type(instance), (Attribute,)):
+
+        for item_name in dir(cls):
```

Comment:
For readability, I prefer inspect.getmembers. I believe they are almost the same thing. Only difference is getmembers doesn't return metaclass attributes. 

The only downside I see is you have to import the inspect module.